torch/lib/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1159,12 +1160,20 @@ c10::intrusive_ptr<ProcessGroup::Work> ProcessGroupNCCL::pointToPoint(
   syncStreams(devices, ncclEvents_[key], ncclStreams_[key]);
 
   // Work itself will create the CUDA events on all GPUs of tensors
-  auto work = initWork(devices, rank_, opType);
+  bool can_profile = tensors.size() == 1;
+  auto work = initWork(
```

Comment:
nit and this is prior to this PR: looks like `initWork` didn't follow clang-format convention? (the last arg line should be split to two). Could you please run a clang-format in a followup PR? 

```
c10::intrusive_ptr<ProcessGroupNCCL::WorkNCCL> ProcessGroupNCCL::initWork(
    std::vector<at::Device> devices,
    int rank,
    OpType opType,
    const char* profilingTitle, const c10::optional<std::vector<at::Tensor>>& inputs) {
  return c10::make_intrusive<ProcessGroupNCCL::WorkNCCL>(devices, rank, opType, profilingTitle, inputs);
}
```